pkg/distributors/contracts/MultiRewards.sol

```diff
@@ -81,6 +83,10 @@ contract MultiRewards is IMultiRewards, IDistributor, ReentrancyGuard, Temporari
         _;
     }
 
+    function setRewardsScheduler(address rs) public onlyOwner {
+        rewardsScheduler = rs;
+    }
```

Comment:
If the scheduler is immutable (and I think it should be immutable - I can't imagine much of a reason why we'd want ot upgrade and I'd avoid this extra complexity), what we could do instead is the following, which will be simpler and cheaper:

```
contract MultRewards
  constructor() {
    scheduler = new Scheduler();
  }
  ...
}

contract Scheduler {
  constructor() {
    multirewards = msg.sender;
  }
}
```

Reply:
(by the way, did you mean to make this mutable on purpose? If so, I'd be interested in hearing the rationale. And in that case we should at least emit an event).

Reply:
Yes, I figured we should leave ourselves the flexibility to upgrade the scheduler (to better handle recurring rewards, for example).  I guess, as part of that, we would want to allow people to cancel scheduled rewards under any conditions as part of upgradeability, as (scheduled rewards currently can't be cancelled once they are scheduled to start).

If we opt for immutable reward schedulers then the deployment pattern above makes sense